src/e-build.js

```diff
@@ -118,7 +118,24 @@ program
         target = targets['default'];
       }
 
-      runNinja(config, target, options.goma, ninjaArgs);
+      try {
+        runNinja(config, target, options.goma, ninjaArgs);
+      } catch (ex) {
+        if (target === targets['node:headers']) {
+          // Older versions of electron use a different target for node headers so try that if the new one fails.
+          try {
+            const olderTarget = 'third_party/electron_node:headers';
+            console.info(
+              `${color.info} Error building ${target}; trying older ${olderTarget} target`,
+            );
+            runNinja(config, olderTarget, options.goma, ninjaArgs);
+          } catch (ex) {
+            tnrow(ex);
+          }
```

Comment:
If this catch block is just going to throw the error, probably don't need it.

Reply:
doh!  Good point